### ISSUE TITLE
Quantum Circuit Learning with Gradient Descent Using Backpropagation

### ISSUE THREAD

**Issue by kenjikun** [arxivtimesquantum/arxivTimesQuantum#18]
## 一言でいうと
量子回路学習(QCL)<sup>1</sup>において，損失関数の計算の際，従来のニューラルネットワークと同様の逆誤差伝搬計算を行えるようにした．提案手法はSPSA,有限要素法に比べ，計算時間が大幅に減ることを数値的に示した．5qubit程度，depthが10程度(下図参照)でも数百倍の高速化が得られている．

![image](https://user-images.githubusercontent.com/23494258/68078030-f7fd9100-fe11-11e9-94b8-b19671b93e38.png)
![image](https://user-images.githubusercontent.com/23494258/68078032-0186f900-fe12-11e9-9f61-b967b53d439b.png)

1: https://arxiv.org/abs/1803.00745
### 論文リンク
https://arxiv.org/abs/1910.14266
### 著者/所属機関
Masaya Watabe1
, Kodai Shiba1,3, Masaru Sogabe3
, Katsuyoshi Sakamoto1,2,
Tomah Sogabe1,2,3*
 Engineering department, The University of Electro-Communications, Tokyo, Japan 2 i-PERC, The University of Electro-Communications, Tokyo, Japan 3 Grid, Inc. Tokyo, Japan
### 投稿日付(yyyy/MM/dd)
2019/10/31
## 手法
![image](https://user-images.githubusercontent.com/23494258/68077983-fda6a700-fe10-11e9-85d4-944901bc2ad5.png)
## 結果
![image](https://user-images.githubusercontent.com/23494258/68077987-09926900-fe11-11e9-8dae-e827ca269314.png)
